codalab/worker/bundle_service_client.py

```diff
@@ -43,6 +44,21 @@ def wrapper(*args, **kwargs):
     return decorator
 
 
+def ensure_str(response):
+    """
+    Ensure the data type of input response to be string
+    :param response: a response in bytes or string
+    :return: the input response in string
+    """
+    if isinstance(response, str):
+        return response
+    try:
+        response = response.decode()
```

Comment:
You can just do `return response.decode()` here

Reply:
Updated!